packaging_automation/postgres-matrix.yml

```diff
@@ -0,0 +1,12 @@
+name: Postgres Version Matrix
+project_name: citus # alternatives: citus, citus-enterprise,pg-auto-failover, pg-auto-failover-enterprise
+# There will be one configuration like this for each project in packaging repo
+# i.e. in all-citus, all-enterprise,all-pgautofailover,all-pgautofailover-enterprise, pgxn-citus etc.
+version_matrix:
+  - 9.4:
```

Comment:
Minor versions are listed here. I assumed that postgres support does not change in patch versions. If it is not, I can add another sub item for patch release

Reply:
`V10.1.1` dropped support for PG11.

Therefore, `v10.1.0` is the single `10.1.x` release with PG11 support. We will not create any such packages, so this matrix is valid as is.

We plan to add PG14 to the build matrix for 10.2. This is currently WIP, so no need to add that here now.

Reply:
Added new items into matrix

Reply:
Sorry for misleading comments above. 10.1.1 was the first enterprise version that dropped PG11 support, nly because 10.1.0 was not released on enterprise.

The current matrix is ok as it is.